### PR TITLE
Machine updates

### PR DESCRIPTION
**Pull request by NinaWie**
So, there existed a branch machine_updates, where you commited some data changes, and then I pulled that branch and added the retirement of two machines. But at the same time, there was an intermediate commit of a new machine to the main branch. How could that happen? I now resolved the conflict, so it should be fine now, but I'm confused why the system didn't work

### CODE REVIEW

File: data/server_locations.json [hunk 1]
```diff
@@ -4884,6 +4884,52 @@
...
+                    33.7706534
+                ]
+            },
+            "properties": {
+                "name": "Six Flags Over Georgia",
+                "active": false,
+                "area": "Georgia",
+                "address": "275 Riverside Parkway SW, Austell",
+                "status": "retired",
+                "external_url": "http://209.221.138.252/Details.aspx?location=1907",
```
Comment (jannisborn) [inline data/server_locations.json:4903]: This seems to be a multimachine -- are we sure they are all retired?

Comment (NinaWie) [reply data/server_locations.json:4903]: The e-mail was "Just visited over the weekend, could not find any machines throughout the park. The penny collectors website stated the machines were removed last year. " and the website also looks like it

**Comment by jannisborn** [comment]
This happened because a user submitted a new machine *via the old system* which created a new branch and opened a PR. That PR was merged, hence the conflict. These situations will start to disappear as users update their app

**Review by jannisborn — APPROVED** [review]
The change in `all_locations.json` is because the machine was a duplication

**Pull request merged**